### PR TITLE
Fixes toolbelts holding everything

### LINKED ISSUE #37186: Storage requirements are broken
Revision b7af6c169b0d358bb37aae64cc89ab7649573415

Anything smallish fits in the RPED, power cells and materials fit in toolbelts, etc.

@kevinz000

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
fix: Fixes toolbelts being able to hold everything.
/:cl:

[why]: Fixes #37186 partially, the rped part is fixed by the rped PR.
>ulility
@kevinz000

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	item_state = "utility"
 	content_overlays = TRUE
 
-/obj/item/storage/belt/ulility/ComponentInitialize()
+/obj/item/storage/belt/utility/ComponentInitialize()
 	. = ..()
 	GET_COMPONENT(STR, /datum/component/storage)
 	STR.can_hold = typecacheof(list(
```
Comment (vuonojenmustaturska) [inline code/game/objects/items/storage/belt.dm]: Assuming that none of the subtypes override can_hold, which they don't seem to be doing, we can save handful of memory and redundant typecache creation by sharing a list like 
```
var/static/list/can_hold = typecacheof(list( etc...
STR.can_hold = can_hold
```

**Review by vuonojenmustaturska — DISMISSED** [review]

**Comment by DaxDupont** [comment]
@vuonojenmustaturska done

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -34,10 +34,10 @@
 	item_state = "utility"
 	content_overlays = TRUE
 
-/obj/item/storage/belt/ulility/ComponentInitialize()
+/obj/item/storage/belt/utility/ComponentInitialize()
```
Comment (Jalleo) [inline code/game/objects/items/storage/belt.dm]: ....... The heck did I not notice that. Ugh.

Comment (Cyberboss) [reply code/game/objects/items/storage/belt.dm]: dude, i'm with you

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**